docs/data-sources/cloud_user_team_assignment.md

```diff
@@ -0,0 +1,75 @@
+To manage such users with this resource, refer to our [migration guide]<link-to-migration-guide>.
+
+## Example Usages
+
+```terraform
+resource "mongodbatlas_cloud_user_team_assignment" "example" {
+  org_id  = var.org_id
+  team_id = var.team_id
+  user_id = var.user_id
+}
+
+data "mongodbatlas_cloud_user_team_assignment" "example_user_id" {
+  org_id  = var.org_id
+  team_id = var.team_id
+  user_id = var.user_id
+}
+
+data "mongodbatlas_cloud_user_team_assignment" "example_username" {
+  org_id   = var.org_id
+  team_id  = var.team_id
+  username = var.user_email
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `org_id` (String) Unique 24-hexadecimal digit string that identifies the organization that contains your projects. Use the [/orgs](#tag/Organizations/operation/listOrganizations) endpoint to retrieve all organizations to which the authenticated user has access.
+- `team_id` (String) Unique 24-hexadecimal digit string that identifies the team to which you want to assign the MongoDB Cloud user. Use the [/teams](#tag/Teams/operation/listTeams) endpoint to retrieve all teams to which the authenticated user has access.
+
+### Optional
+
+- `user_id` (String) Unique 24-hexadecimal digit string that identifies the MongoDB Cloud user.
+- `username` (String) Email address that represents the username of the MongoDB Cloud user.
+
+### Read-Only
+
+- `country` (String) Two-character alphabetical string that identifies the MongoDB Cloud user's geographic location. This parameter uses the ISO 3166-1a2 code format.
+- `created_at` (String) Date and time when MongoDB Cloud created the current account. This value is in the ISO 8601 timestamp format in UTC.
+- `first_name` (String) First or given name that belongs to the MongoDB Cloud user.
+- `invitation_created_at` (String) Date and time when MongoDB Cloud sent the invitation. MongoDB Cloud represents this timestamp in ISO 8601 format in UTC.
+- `invitation_expires_at` (String) Date and time when the invitation from MongoDB Cloud expires. MongoDB Cloud represents this timestamp in ISO 8601 format in UTC.
+- `inviter_username` (String) Username of the MongoDB Cloud user who sent the invitation to join the organization.
+- `last_auth` (String) Date and time when the current account last authenticated. This value is in the ISO 8601 timestamp format in UTC.
+- `last_name` (String) Last name, family name, or surname that belongs to the MongoDB Cloud user.
+- `mobile_number` (String) Mobile phone number that belongs to the MongoDB Cloud user.
+- `org_membership_status` (String) String enum that indicates whether the MongoDB Cloud user has a pending invitation to join the organization or they are already active in the organization.
+- `roles` (Attributes) Organization and project level roles to assign the MongoDB Cloud user within one organization. (see [below for nested schema](#nestedatt--roles))
```

Comment:
we don't often use "attributes" in the docs. Is this an object / list of objects?